libs/perun/services/src/lib/gui-auth-resolver.service.ts

```diff
@@ -89,82 +89,101 @@ export class GuiAuthResolver {
   canManageFacilities(): boolean {
     return this.hasAtLeastOne(
       Role.PERUNADMIN,
+      Role.PERUNADMINBA,
       Role.PERUNOBSERVER,
       Role.FACILITYADMIN,
       Role.FACILITIYOBSERVER
     );
   }
 
   isPerunAdmin(): boolean {
-    return this.principalRoles.has(Role.PERUNADMIN);
+    return this.principalRoles.has(Role.PERUNADMIN) || this.principalRoles.has(Role.PERUNADMINBA);
   }
 
   isPerunAdminOrObserver(): boolean {
-    return this.hasAtLeastOne(Role.PERUNADMIN, Role.PERUNOBSERVER);
+    return this.hasAtLeastOne(Role.PERUNADMIN, Role.PERUNOBSERVER, Role.PERUNADMINBA);
   }
 
   isVoAdmin(): boolean {
-    return this.hasAtLeastOne(Role.PERUNADMIN, Role.VOADMIN);
+    return this.hasAtLeastOne(Role.PERUNADMIN, Role.VOADMIN, Role.PERUNADMINBA);
   }
 
   isThisVoAdminOrObserver(id: number): boolean {
     return (
       this.editableVos.includes(Number(id.toString())) ||
       this.observableVos.includes(Number(id.toString())) ||
-      this.hasAtLeastOne(Role.PERUNADMIN, Role.PERUNOBSERVER)
+      this.hasAtLeastOne(Role.PERUNADMIN, Role.PERUNOBSERVER, Role.PERUNADMINBA)
     );
   }
 
   isThisVoAdmin(id: number): boolean {
-    return this.editableVos.includes(id) || this.principalRoles.has(Role.PERUNADMIN);
+    return (
+      this.editableVos.includes(id) ||
+      this.principalRoles.has(Role.PERUNADMIN) ||
+      this.principalRoles.has(Role.PERUNADMINBA)
+    );
   }
 
   isGroupAdmin(): boolean {
-    return this.hasAtLeastOne(Role.PERUNADMIN, Role.GROUPADMIN);
+    return this.hasAtLeastOne(Role.PERUNADMIN, Role.GROUPADMIN, Role.PERUNADMINBA);
   }
 
   isOnlySponsor(): boolean {
     return this.hasAtLeastOne(Role.SPONSOR);
   }
 
   isThisGroupAdmin(id: number): boolean {
-    return this.editableGroups.includes(id) || this.principalRoles.has(Role.PERUNADMIN);
+    return (
+      this.editableGroups.includes(id) ||
+      this.principalRoles.has(Role.PERUNADMIN) ||
+      this.principalRoles.has(Role.PERUNADMINBA)
```

Comment:
Just a thought - how about replacing all of these checks for PERUNADMIN with the call of `isPerunAdmin()`. This way should we ever remove `PERUNADMINBA` or add another similar role, the number of necessary changes will be minimal.

Reply:
I replaced it as much as I could, please check, if I haven't missed anything.

Reply:
I think you got all of it. Have you also considered adding some readable text for the role in `en.json`? (the roles page component uses this)